src/stackdriver-nozzle/scripts/firehose-to-stdout.go

```diff
@@ -0,0 +1,26 @@
+package main
```

Comment:
not a hard stance here but i think this one may be appropriate as a part of the main binary. That said we only expect these to get ran by bosh where someone won't use that output so feel free to ignore it.